saleor/webhook/payloads.py

```diff
@@ -408,11 +425,16 @@ def generate_product_variant_payload(product_variants: Iterable["ProductVariant"
             "channel_listings": lambda v: json.loads(
                 generate_product_variant_listings_payload(v.channel_listings.all())
             ),
+            "quantity_in_stocks": lambda v: generate_product_variant_stocks_payload(v),
```

Comment:
This change is not mentioned in the changelog.

Reply:
This field misses information about allocations. Are you sure that is proper behavior?

Reply:
This field is also returning one value for all warehouses are we sure that is proper behavior?